docs/source/repositories.rst

```diff
@@ -1,9 +1,21 @@
 Repositories
-=====
-NEXT software is structed in two main repositories:
+===============
+NEXT software can be found on Github as `next-exp <https://github.com/next-exp/>`_. The main code is structured in two main repositories:
+
+* :doc:`NEXUS`: Simulation tool in charge of the detector geometries and particle generators
+* :doc:`IC`: Reconstruction tool for detector data and simulations
+
+Other reconstruction tools can be found there (as :doc:`ICAROS`, :doc:`olivia`) that work as a support to the main software code.
+Any NEXT repository can be forked into your own Github account. :doc:`workflow` explains in detail the way users and developers can make us of these repositories.
```

Comment:
make us --> make use. Maybe simply using `use` makes the sentence more readable.